### ISSUE TITLE
York Region, Ontario, Canada is missing city and district data

### ISSUE THREAD

**Issue by Valachio** [openaddresses/openaddresses#4295]
[York Region](http://results.openaddresses.io/sources/ca/on/york) is missing city and district data.

District names are important for certain cities in York Region, namely Vaughan and Markham.  The locals tend to use the district name to refer to their town.

**Comment by justinelliotmeyers** [comment]
@Valachio is this an official part of the address? I know sometimes we have city names in the data and sometimes they are emoty. I'm wondering if York formally has this; sometimes the municipalities don't specify, and sometimes they do. Just curious. Thanks!!

**Comment by justinelliotmeyers** [comment]
does this work for you @Valachio http://insights-york.opendata.arcgis.com/datasets/address-points

**Comment by Valachio** [comment]
@justinelliotmeyers Yes that link is where I fetched my data from originally.  There should be both district and city names - those fields are in that file you linked.
